src/web/lib/components/TermsPrivacyModal/index.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
```

Comment:
Made this into it's own component to keep the App component uncluttered with strings